pkg/provider/kubernetes/crd/fixtures/with_middleware_crossprovider.yml

```diff
@@ -0,0 +1,47 @@
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: stripprefix
+  namespace: default
+
+spec:
+  stripPrefix:
+    prefixes:
+      - /tobestripped
+
+---
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: addprefix
+  namespace: foo
+
+spec:
+  addPrefix:
+    prefix: /tobeadded
+
+---
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: test2.crd
+  namespace: default
+
+spec:
+  entryPoints:
+    - web
+
+  routes:
+    - match: Host(`foo.com`) && PathPrefix(`/tobestripped`)
+      priority: 12
+      kind: Rule
+      services:
+        - name: whoami
+          port: 80
+      middlewares:
+        - name: stripprefix
+        - name: addprefix
+          namespace: foo
+        - name: file@basicauth
```

Comment:
Shouldn't this be `basicauth@file` so it adheres to the `middlewarename@provider` format? Same for line below.

Reply:
indeed

Reply:
done